manifests/v/VMware/WorkstationPro/16.2.3.21887/VMware.WorkstationPro.installer.yaml

```
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.1.1 $debug=QUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: VMware.WorkstationPro
+PackageVersion: 16.2.3.21887
```

```suggestion
PackageVersion: 16.2.3
```
This must match Control Panel.

---

You will also need to update the version folder from `16.2.3.21887` to `16.2.3`